blockchain/upgrade.go

```diff
@@ -428,6 +428,7 @@ func deserializeUtxoEntryV0(serialized []byte) (map[uint32]*UtxoEntry, error) {
 			blockHeight: int32(blockHeight),
 			packedFlags: packedFlags,
 		}
+		entries[outputIndex].pkScript, _ = entries[outputIndex].tokenData.SeparateTokenDataFromPKScriptIfExists(entries[outputIndex].pkScript, 0)
```

Comment:
Noticed every time we call this method, we are masking the error. There are a number of ways for this to fail, is this intentional?

Reply:
Modified it.